algorithms/MDLMDec/mgf_to_parquet.py

```
@@ -0,0 +1,224 @@
+            'precursor_mass': [],
+            'mz_array': [],
+            'intensity_array': [],
+        }
+        md = gather_file_md(file)
+        counter += len(md)
+        for a, dic in md.items():
+            mzs = []
+            Abs = []
+            with open(file) as f:
+                f.seek(md[a]['pos'])
+                for i in range(md[a]['nmpks']):
```

Ruff (0.16.1)

[warning] Loop control variable `i` not used within loop body

(B007)